doc/neps/ufunc-overrides.rst

```diff
@@ -620,8 +620,8 @@ make more sense to ask classes like ``MyObject`` to implement a full
 ``__array_ufunc__`` [6]_. In the end, allowing classes to opt out was
 preferred, and the above reasoning led us to agree on a similar
 implementation for :class:`ndarray` itself. We decided to require disabling
-ufuncs to opt out to ensure that a class cannot define Ufuncs to return
-different results than the corresponding binary operations (i.e., if
+ufuncs to opt out in order to ensure that a class cannot define Ufuncs to
```

Comment:
Hmm, now that I look at this in combination with "We decided to require disabling" I still can't figure out what it says ;) I think what this is trying to say is that ufuncs will raise TypeError when one of the arguments opts out in order to ensure ...